graphql_api/types/account/account.graphql

```diff
@@ -3,4 +3,12 @@ type Account {
   oktaConfig: OktaConfig
   totalSeatCount: Int!
   activatedUserCount: Int!
+  organizations(
+    ordering: AccountOrganizationOrdering
+    orderingDirection: OrderingDirection
+    first: Int
+    after: String
+    last: Int
+    before: String
+  ): AccountOrganizationConnection! @cost(complexity: 25, multipliers: ["first", "last"])
```

Comment:
complexity is same as on user organizations list. Seems to make sense to keep things consistent.